spec/GraphQLOverHTTP.md

```diff
@@ -193,10 +193,10 @@ And for a _GraphQL response_:
 For details of the shapes of these JSON payloads, please see
 [Request](#sec-Request) and [Response](#sec-Response).
 
-If the media type in a `Content-Type` or `Accept` header includes encoding
-information, then the encoding MUST be `utf-8` (e.g.
-`Content-Type: application/graphql-response+json; charset=utf-8`). If encoding
-information is not included then `utf-8` MUST be assumed.
+If the media type in a `Content-Type` or `Accept` header does not
+include encoding information, then `utf-8` MUST be assumed (e.g.
+`Content-Type: application/graphql-response+json; charset=utf-8`).
```

Comment:
This example doesn't seem to fit any more
```suggestion
include encoding information, then `utf-8` MUST be assumed (e.g.
for header `Content-Type: application/graphql-response+json`, UTF-8 encoding would be assumed).
```

Reply:
:+1: